portfolio/src/main/webapp/index.html

```diff
@@ -37,7 +37,30 @@ <h2 id="titles">
                     <a class="button1" href="https://github.com/AbakirH">
                         <i class="fa fa-github fa-5x pad" aria-hidden="true"></i>
                     </a>
+                </div>
             </div>
         </div>
+        
+    <div id="projects">
+        <h1>Projects</h1>
+        <div id='project1' >
+            <img class='projectIMG' src="Images/pokemonGame1.png">
+            
+            <div id='section1'>
+              <h2>Pokemon:
+                  <span>Developed in All Star Code</span>
+              </h2>
+              <p class='lead1'>This is a simple Pokemon simulator battle game
+              </p>
+              <div class='link'>
+                    <button class=' button' onclick="viewGame()">View the Code</button>                
```

Comment:
Not sure if this is intentional, but there's a space before "button" here. This may also affect the styling effects for this css selector.

Reply:
okay I will fix that, thank you